spec/features/create_customer_by_transaction_spec.rb

```diff
@@ -33,6 +33,8 @@ def create_transaction
     sign_in user
   end
 
+  after { Capybara.reset_sessions! }
```

Comment:
@dissident do you really need it in every file? May be it will be better to move in support? (see database cleaner for example)